lesson_2/home_work2_6.py

```diff
@@ -0,0 +1,6 @@
+group_of_people = ["John", "Rebecca", "Alessandro", "Melena", "John"]
+uniq_names = []
+for names in group_of_people:
+    if names not in uniq_names:
+       uniq_names.append(names)
+print(uniq_names)
```

Comment:
as I mentions in the lesson this HW should be done only using functions of types which were shown

Reply:
I have changed - but I do not see changes now((